greenmail-core/src/main/java/com/icegreen/greenmail/imap/commands/SearchCommandParser.java

```diff
@@ -34,6 +35,8 @@ public SearchTerm searchTerm(ImapRequestLineReader request)
             throws ProtocolException {
         SearchTerm resultTerm = null;
         SearchTermBuilder b = null;
+        SearchKey key = null;
+	boolean Orkey = false;
```

Comment:
Java coding convention : attributes start with lower case and use camel case: "orKey"